_data/Sakshigupta06/sakshigupta06.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Sakshi Gupta
+institution: Galgotias University
+profile_pic: sakshii.jpg
```

Comment:
Hey @sakshigupta06 , the profile_pic link should be exact as per the filename ( with extension ) . So `jpg` -> `png`